client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -71,6 +72,7 @@ class GoogleAnalyticsForm extends Component {
 			jetpackVersionSupportsModule,
 			showUpgradeNudge,
 			site,
+			siteId,
```

Comment:
Sure! However, I did it this way on purpose, for several reasons:

* `siteId` is already retrieved by using the `getSelectedSiteId` selector, so we just add it to `mapStateToProps`.
* I prefer using `getSelectedSiteId` instead of `site.ID` because it's separately tested and is bulletproof in case `site` hasn't been loaded yet.

Of course, if you feel `site.ID` is better, we can refactor it, but we'll also have to do it everywhere else where `siteId` is used.

Thanks for the review 👍